extensions/wikia/LookupContribs/SpecialLookupContribs_ajax.php

```diff
@@ -131,12 +131,15 @@ public static function prepareLookupUserData( $activityData, $username ) {
 
 		$rows = [];
 		foreach ( $activityData as $row ) {
+			$discussionActivity = LookupUserPage::fetchDiscussionPostCountAndDate($username, $row['id']);
 			$rows[] = [
 				$row['id'], // wiki Id
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)